Character.cpp

```diff
@@ -5,6 +5,36 @@ Character::Character(const std::string characterName, const unsigned int healthP
 	HP = healthPoints;
 }
 
+Character Character::parseUnit(std::string fileName)
+{
+	std::ifstream sourceFile(fileName);
+	if (sourceFile.good())
+	{
+		fileName.resize(fileName.length() - 5);
+
+		/*std::string command = "\
+		python json2txt.py \
+		";
+
+		command.append(fileName);
+		system(command.c_str());*/
+		std::ifstream outputFile(fileName + ".txt");
```

Comment:
ifstream otuputfile-ra?!

Reply:
A név arra utal, hogy a konverzió végterméke ez a fájl. A python scriptben logikus ez az elnevezés, és az uniformitás miatt használtam itt is. Át lesz nevezve.